packages/nexus-sdk/src/Archive/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Fetchers } from '../types';
+import { NexusContext } from '../nexusSdk';
+import { ArchivePayload, GetArchiveOptions } from './types';
+import { buildQueryParams } from '../utils';
+
+const Archive = (
+  { httpGet, httpPut, httpPost, poll }: Fetchers,
+  context: NexusContext,
+) => {
+  return {
+    get: (
+      orgLabel: string,
+      projectLabel: string,
+      archiveId: string,
+      options?: GetArchiveOptions,
+    ): Promise<string> => {
```

Comment:
I think the Promise type is more something like:

```suggestion
    ): Promise<Archive | Blob | string> => {
```